marquez_airflow/dag.py

```diff
@@ -74,7 +75,7 @@ def create_dagrun(self, *args, **kwargs):
             # TODO: Use 'anonymous' owner for now, but we may want to use
             # the 'owner' attribute defined via default_args for a DAG
             marquez_client.create_namespace(self.marquez_namespace,
-                                            "anonymous")
+                                            "datakin")
```

Comment:
We want to keep the user `anonymous`.

Reply:
Yes and it is temporary. Local testing needs a git commit sha. Hence this commit. Before this PR goes off from Draft, I will revert it back.